examples/sources.rs

```diff
@@ -1,3 +1,6 @@
+fn main() {}
+
+/*
```

Comment:
TODO: revert this before merging
I comment this out because I get compilation errors in my editor because etherscan rate limits me.

Reply:
You can log in to etherscan and setup an API key :smile: